docs/source/user-guide.md

```diff
@@ -247,7 +247,10 @@ explanation.save("output_path", "name")
 Black-box mode does not update the model (treating the model as a black box).
 Black-box approaches are based on the perturbation of the input data and measurement of the model's output change.
```

Comment:
I would add:
` For black-box mode we support 2 algorithms: **AISE** (by default) and [**RISE**](https://arxiv.org/abs/1806.07421). First is more effective for generating saliency maps for a few specific classes. The second one - to generate maps for all classes at once.`